2.2/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM openshift/base-centos7
+
+# This image provides a Ruby 2.2 environment you can use to run your Ruby
+# applications.
+
+MAINTAINER Jakub Hadvig <jhadvig@redhat.com>
```

Comment:
Yeah it would make sense to change it for rhel images(but keep the MAINTAINER) with some support mail address. Spoke to @bparees regarding that and we will come with answer